cookbooks/ey-mysql/resources/mysql_slave.rb

```diff
@@ -106,6 +106,14 @@
     only_if { !Dir.glob("#{node['mysql']['datadir']}/slave-relay*").empty? && !volume_from_slave_snapshot }
   end
 
+  # the master writes it's uuid to <datadir>/auto.cnf, the slave needs that removed so it will gen it's own
+  execute "remove auto.cnf" do
```

Comment:
Should use File resource rather than executing shell

Reply:
Updated to use file resource ad tested it working as expacted:

```
[2022-09-02T10:43:47+00:00] INFO: removing <datadir>/auto.cnf
[2022-09-02T10:43:47+00:00] INFO: removing <datadir>/auto.cnf
    * file[/db/mysql/5.7/data//auto.cnf] action delete    * file[/db/mysql/5.7/data//auto.cnf] action delete[2022-09-02T10:43:47+00:00] INFO: Processing file[/db/mysql/5.7/data//auto.cnf] action delete (ey-mysql::slave line 109)
```